alembic/templates/generic/env.py

```diff
@@ -25,6 +30,12 @@
 # my_important_option = config.get_main_option("my_important_option")
 # ... etc.
 
+alembic_url = os.getenv(
+    "ALEMBIC_URL", "postgresql+psycopg2://postgres:postgres@localhost:5432/postgres"
```

Comment:
The general template should not assume any url

Reply:
got it